common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalBlockStoreClient.java

```diff
@@ -256,6 +256,20 @@ public void onFailure(Throwable e) {
     }
   }
 
+  @Override
+  public boolean removeShuffleMerge(String host, int port, int shuffleId) {
+    checkInit();
+    try {
+      TransportClient client = clientFactory.createClient(host, port);
```

Comment:
do we expect to take any action after sending this message via a callback? If so we should consider using `sendRpc` instead of `send`.